UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/FunctionalTests/POSAIOValidFunctionalScenarios.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Net;
+using FluentAssertions;
+using NUnit.Framework;
+using UKHO.PeriodicOutputService.API.FunctionalTests.Helpers;
+using static UKHO.PeriodicOutputService.API.FunctionalTests.Helpers.TestConfiguration;
+
+namespace UKHO.PeriodicOutputService.API.FunctionalTests.FunctionalTests
+{
+    [Category("POSAIOValidFunctionalScenarios")]
+    public class POSAIOValidFunctionalScenarios
+    {
+        private static readonly POSWebJobApiConfiguration posWebJob = new TestConfiguration().POSWebJobConfig;
+        private static readonly POSFileDetails posDetails = new TestConfiguration().posFileDetails;
+        private static readonly FSSApiConfiguration FSSAuth = new TestConfiguration().FssConfig;
+        private string DownloadedFolderPath { get; set; }
+        private string fssJwtToken;
+
+        [OneTimeSetUp]
+        public async Task Setup()
+        {
+            HttpResponseMessage apiResponse = MockHelper.ConfigureFMAio(posWebJob.MockApiBaseUrl, posWebJob.FMConfigurationValidAIOProductIdentifier);
+            apiResponse.StatusCode.Should().Be((HttpStatusCode)200);
+            await CommonHelper.RunWebJobAio();
+        }
+
+        [Test]
+        public async Task WhenICallBatchDetailsEndpointWithValidAioBatchId_ThenBatchDetailsShouldBeCorrect()
+        {
+            HttpResponseMessage apiResponse = await GetBatchDetails.GetBatchDetailsEndpoint(FSSAuth.BaseUrl, posDetails.AIOValidBatchId);
+            apiResponse.StatusCode.Should().Be((HttpStatusCode)200);
+
+            dynamic batchDetailsResponse = await apiResponse.DeserializeAsyncResponse();
+
+            GetBatchDetails.GetBatchDetailsResponseValidationForAio(batchDetailsResponse);
+        }
```

Comment:
where are the tests for confirmation that if the contents are wrong that the process detects correctly whats wrong etc

Reply:
Hi Ray,  w.r.t POS it's just download the AIO.zip from FSS --> creates the .iso & .iso.sha1 files --> Changes the BU  ---> then upload back to FSS. So POS doesn't check the contents of any files/folders. Validating contents of files/folders done in ESS(Automation tests are in place).  Thanks